packages/components/src/toolbar/toolbar-container.native.js

```diff
@@ -9,8 +9,8 @@ import { View } from 'react-native';
 import styles from './style.scss';
 import { withTheme } from '../mobile/dark-mode';
 
-const ToolbarContainer = ( { useStyle, passedStyle, children } ) => (
-	<View style={ [ useStyle( styles.container, styles.containerDark ), passedStyle ] }>
+const ToolbarContainer = ( { useStyle, passedStyle, children, rtl = false } ) => (
```

Comment:
Is this still being used? I only see it changing the style for a border, and I don't see any borders in the toolbar

Reply:
Yes it looks like below

<img src="https://user-images.githubusercontent.com/21242757/66154439-bb295900-e61d-11e9-8800-51391b9bf9d7.png" width="400">

Reply:
I'm not understanding this `rtl` parameter then. From what I see on the web, toolbar has a border on all sides by default, and BlockToolbar modifies that to only have a right border.

I'm not sure why the native variant seems to be doing only the left border.

In any case, when I see `rtl` I would expect that prop to control the direction in when the elements inside the toolbar are laid out. If we want to change just the border, I think it's better to put that in the `passedStyle` prop directly.

Reply:
There is another case that I need to revert those changes with `rtl` because of opacity mentioned by @iamthomasbishop so I need to provide own pipe component